src/services/utility-helpers.js

```diff
@@ -85,6 +85,27 @@ export function timeToHHmmss(secTime, showHrs = false, showMs = false) {
   return timeStr;
 }
 
+/**
+ * Convert a given time in seconds to a string read as a human, these
+ * are used in structure navigation to convey timestamps associated with
+ * media-fragments in a more presentable way for assistive technology tools.
+ * @function Utils#screenReaderFriendlyTime
+ * @param {Number} time time in seconds
+ * @returns {String} time string read as a human
+ */
+export function screenReaderFriendlyTime(time) {
+  const hhmmssTime = timeToHHmmss(time, true, true);
+  if (hhmmssTime != '') {
+    const [hours, minutes, seconds] = hhmmssTime.split(':');
+    let screenReaderTime = parseFloat(hours) > 0 ? `${parseFloat(hours)} hours ` : '';
```

Comment:
What do you think about making these labels be singular when they are only 1 (e.g. 1 hour 1 minute 1 second)?

Also what is expected behavior for 1:00:34?  I think this would currently read "1 hours 34 seconds".  Would it be better as "1 hour 0 minutes 34 seconds"

Reply:
Yes, that makes more sense. I'll make that change 👍